eng/build-job.yml

```diff
@@ -72,9 +72,21 @@ jobs:
         value: $(buildConfigUpper)
     - name: officialBuildIdArg
       value: ''
+    - name: ibcOptimizeArg
+      value: ''
     - ${{ if and(eq(variables['System.TeamProject'], 'internal'), ne(variables['Build.Reason'], 'PullRequest')) }}:
       - name: officialBuildIdArg
         value: '-officialbuildid=$(Build.BuildNumber)'
+      # IBCMerge is currently Windows-only and x86/x64-only
+      - ${{ if and(eq(parameters.osGroup, 'Windows_NT'), or(eq(parameters.archType, 'x64'), eq(parameters.archType, 'x86'))) }}:
```

Comment:
@adiaaida we do want IbcOptimize set for Linux also, right?  This looks like it only gets enabled for Windows_NT x64 and x86 (if I'm getting my parenthesis right).

Reply:
Also, IbcOptimize should only be specified for release builds.

Reply:
IbcOptimize here will only set it for official builds.

Reply:
As discussed above, we have to do something different for IBC on linux, namely that we have to bring the un-crossgened assemblies over to windows to run IBCMerge, then then send them back to linux to crossgen. @jkoritzinsky said he would do the refactoring for Linux in another PR (here: https://github.com/dotnet/coreclr/pull/22843#discussion_r260388422)

Reply:
Gotcha.  Thanks, I missed that.